platform/windows/display_server_windows.cpp

```diff
@@ -1119,7 +1119,7 @@ bool DisplayServerWindows::window_can_draw(WindowID p_window) const {
 
 	ERR_FAIL_COND_V(!windows.has(p_window), false);
 	const WindowData &wd = windows[p_window];
-	return wd.minimized;
+	return !wd.minimized;
```

Comment:
This seems unrelated to the rest of the PR. Although the original does look wrong, was this connected to a bug you noticed?

Reply:
Yes, this is a typo and prevented rendering the frames as fast as possible when disabling v-sync on Windows.

Reply:
This should probably go in its own PR then to be reviewed and merged independently of this one.

Reply:
I could do that, but is that really necessary for this tiny change since it is a bug that prevented the v-sync setting to work properly on Windows?

Reply:
It looks to me like a bug that would also affect Godot on Windows regardless of this PR, no? 
This method is used in `Window::can_draw()` and in `main.cpp`:
```
OS::get_singleton()->add_frame_delay(DisplayServer::get_singleton()->window_can_draw());
```
So it has potential for changing behavior for Windows users significantly (hopefully for the better), and thus it's better if it's merged independently so that its effect can be assessed independently of this PR for VSync.

Of course you need the fix here so you can also keep the change here and rebase once the other PR is merged (which is likely to come sooner than this here PR given the difference in complexity of the changes).

Reply:
Ok, then I will split this up.

Reply:
Opened  #49253